_playwright-tests/UI/SnapshotErrataCountAndFilter.spec.ts

```diff
@@ -104,7 +100,10 @@ test.describe('Snapshot Errata Count and Filter', () => {
 
       await expect(page.getByRole('button', { name: 'Clear filters' })).toBeVisible();
 
-      await expect(page.getByTestId('errata_table')).toBeVisible();
+      await expect(snapshotListModal.getByRole('table', { name: 'errata table' })).toBeVisible({
```

Comment:
this should work to target the table: `await expect(snapshotListModal.getByLabel('errata table')).toBeVisible();`

Reply:
OK, thank you